tensorflow_addons/layers/optical_flow.py

```diff
@@ -82,8 +82,8 @@ def correlation_cost(input_a,
       A `Tensor` of the format specified by `data_format`.
     """
 
-    with tf.name_scope(name or "correlation_cost"):
-        op_call = _correlation_cost_op_so.correlation_cost
+    with tf.name_scope(name or "_correlation_cost"):
+        op_call = _correlation_cost_op_so._correlation_cost
```

Comment:
It seems _correlation_cost_op_so has no attribute '_correlation_cost'. Have you test the code on the local environment. And I believe correlation_cost is more suitable for name scope.

Reply:
Hi @fsx950223 , we won't be able to import it still. I will change the name scope but I am not sure how to go with the test file so as to run the test.
One way is to make correlation_cost nested in some other function and calling the outer function. But this will not be private, however, easy to import in test.
Second way is to add _correlation_cost function in the test file also.
Alternatively, I think, if somehow, correlation_cost function can be added as a private function to the CorrelationCost class and then we can call call it.